tasks.py

```diff
@@ -79,3 +80,23 @@ def update_build_number(ctx, new_build_number):
     assert new_build_number > current_build_number
     with open("modal_version/_version_generated.py", "w") as f:
         f.write(f"{copyright_header_full}\nbuild_number = {new_build_number}\n")
+
+
+@task
+def type_stubs(ctx):
+    # we only generate type stubs for modules that contain synchronicity wrapped types
```

Comment:
couldn't this be inferred automatically if we just scan the whole package?

Reply:
Yeah we could do that. It's a bit nicer to not generate the stubs for the packages where there are no wrapper classes, since in those cases editors would just use the implementation types instead and be able to infer more implicit types, but we could scan the package and check which modules contain/declare synchronicity wrappers I suppose?